features/checkout/order_promotion/order_promotion_integrity_validation.feature

```diff
@@ -12,26 +12,28 @@ Feature: Order promotions integrity
         And there is a promotion "Christmas sale"
         And I am a logged in customer
 
-    @ui
+    @ui @api
     Scenario: Preventing customer from completing checkout with already expired promotion
         Given this promotion gives "$10.00" discount to every order
         And this promotion expires tomorrow
         And I added product "PHP T-Shirt" to the cart
-        And I have proceeded selecting "Offline" payment method
+        And I have specified the billing address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
```

Comment:
```suggestion
        And I specified the billing address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
```
to be consistent with other steps

Reply:
Ok, I can do it but I need to do this update by command and that's why I created a step like this (maybe it's a bad idea). Should I refactor this step on API context to update this by command instead of API call (but I think it's a not good idea)? Or we will leave it as it is? :) I did it like that because of the conflict of contexts in behat steps.